Workbooks/UpdateCompliance/DeviceListAlert/Devices list for Alert Subtype.workbook

```diff
@@ -0,0 +1,236 @@
+            "name": "mappedWorkspace",
+            "label": "workspace",
+            "type": 5,
+            "isRequired": true,
+            "query": "resources\r\n| where type == \"microsoft.operationalinsights/workspaces\"\r\n| where \"{mappedSubscription:subscriptionId}\" == subscriptionId\r\n| project id",
+            "crossComponentResources": [
+              "{mappedSubscription}"
+            ],
+            "value": "",
+            "isHiddenWhenLocked": true,
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "03761a72-d969-426b-ba54-d1e709610b2d",
+            "version": "KqlParameterItem/1.0",
+            "name": "_WorkbookContext",
+            "label": "Workbook Context",
+            "type": 1,
+            "description": "Whether workbook is opened standalone or as part of the Hub.",
+            "isRequired": true,
+            "isHiddenWhenLocked": true,
+            "criteriaData": [
+              {
+                "criteriaContext": {
+                  "operator": "Default",
+                  "resultValType": "static",
+                  "resultVal": "Focused"
+                }
+              }
+            ],
+            "timeContext": {
+              "durationMs": 86400000
+            }
+          },
+          {
+            "id": "f503a201-a9bd-4085-8862-22e2c0735ed4",
+            "version": "KqlParameterItem/1.0",
+            "name": "_SnapshotTime",
+            "label": "Snapshot time",
+            "type": 1,
+            "description": "Time indicating the latest snapshot",
+            "isRequired": true,
+            "query": "UCClient\r\n| where TimeGenerated >= ago(7d)\r\n| summarize max(TimeGenerated)",
+            "crossComponentResources": [
+              "{mappedWorkspace}"
+            ],
+            "isHiddenWhenLocked": true,
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          },
+          {
+            "id": "42e71793-2ac1-46f8-9b8e-3c6a9fedb05b",
+            "version": "KqlParameterItem/1.0",
+            "name": "_AlertSubtype",
+            "label": "AlertSubtype",
+            "type": 1,
+            "description": "AlertSubtype value",
+            "isRequired": true,
+            "isHiddenWhenLocked": true,
+            "value": "DownloadIssue"
+          },
+          {
+            "id": "c821a6fc-40b0-4369-9527-f127599cecac",
+            "version": "KqlParameterItem/1.0",
+            "name": "_TargetBuild",
+            "type": 1,
+            "description": "Target Build value",
+            "isRequired": true,
+            "isHiddenWhenLocked": true,
+            "value": "10.0.19044.1766"
```

Comment:
go over the workbooks and double check the static values for the params, if they aren't needed or are dynamically generated you can just delete the `"value"` field